katdal/test/test_applycal.py

```diff
@@ -66,11 +65,11 @@
 GAIN_EVENTS = list(range(0, N_DUMPS, 10))
 BAD_GAIN_ANT = 3
 BAD_GAIN_DUMPS = [20, 40]
-CAL_PRODUCTS = ['K', 'B', 'G']
+CAL_PRODUCTS = ['cal.K', 'cal.B', 'cal.G']
```

Comment:
This looks different to what's actually used by visdatav4.

Reply:
Sure, but it's a unit test so I figured I won't add indirection yet.

Reply:
So do you have a separate test for indirection somewhere?

Reply:
I'll add something when I rework the default stream.